.circleci/config.yml

```diff
@@ -140,7 +140,7 @@ jobs:
       - run:
           name: Install test dependencies
           command: |
-            sudo apt-get install -y libqt5gui5 --no-install-recommends
+            sudo apt-get install -y libqt5gui5 libxcb-cursor0 --no-install-recommends
```

Comment:
Is this a library that should also be explicitly stated as a depedency of the `.deb`? Or it's so prevalent that we don't need it? Or maybe it's only for building (idk).

Reply:
The debs/rpms do not have PySide6 installed, but PySide2. This library brings all the necessary dependencies.